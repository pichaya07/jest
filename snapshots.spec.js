const user = {
    name:'Tony Tinkertons',
    age:25,
    job:'inventor'
}

test ('user match', () => {
    const userString = "{\"name\":\"Tony Tinkerton\",\"age\":25,\"job\":\"inventor\"}";
    expect(user).toMatchSnapshot()
})