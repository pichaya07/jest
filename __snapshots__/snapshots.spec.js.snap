// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user match 1`] = `
Object {
  "age": 25,
  "job": "inventor",
  "name": "Tony Tinkertons",
}
`;
